src/examples/mfem_common_plane.cpp

```diff
@@ -78,8 +78,8 @@ int main( int argc, char** argv )
   double dt = 0.001;
   // end time of the simulation
   double t_end = 2.0;
-  // kinematic penalty parameter
-  double p_kine = 10000.0;
+  // use const penalty (true) or element penalty (false)
```

Comment:
Does this comment make sense when you set use_const_penalty = false;?

Reply:
I think so.  The example would use element penalty in that case

Reply:
change to (if true) and (if false)